### PR TITLE
Java homework Lecture3/Level2 has been done

### PR DESCRIPTION
**Pull request by bumax1989**

### CODE REVIEW

**Review by anastasiaasadcheva94 — APPROVED** [review]
lgfm

**Review by AleksandrBohan — APPROVED** [review]
lgfm

**Review by KaydunovDenis — APPROVED** [review]

File: homework/java/BuchynskiM_JHL1T2/L3/Level2/Encode.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package Lecture3_Level2;
```
Comment (ArtikTico) [inline homework/java/BuchynskiM_JHL1T2/L3/Level2/Encode.java:1]: rename package

File: homework/java/BuchynskiM_JHL1T2/L3/Level2/Encode.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    public char getSymbol() {
+        return symbol;
+    }
+
+    public void setSymbol(char symbol) {
+        this.symbol = symbol;
+    }
+
+    public void encode(short code){
```
Comment (ArtikTico) [inline homework/java/BuchynskiM_JHL1T2/L3/Level2/Encode.java:31]: add space

File: homework/java/BuchynskiM_JHL1T2/L3/Level2/EncoderDemo.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package Lecture3_Level2;
```
Comment (ArtikTico) [inline homework/java/BuchynskiM_JHL1T2/L3/Level2/EncoderDemo.java:1]: rename package

File: homework/java/BuchynskiM_JHL1T2/L3/Level2/EncoderDemo.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        encoder.encode(encoder.getCode());
+        decoder.decode(decoder.getSymbol());
+
+        decoder.setSymbol('S');
+        encoder.setCode((short)88);
+
+        encoder.encode(encoder.getCode());
+        decoder.decode(decoder.getSymbol());
+    }
+}
```
Comment (ArtikTico) [inline homework/java/BuchynskiM_JHL1T2/L3/Level2/EncoderDemo.java:17]: prove that it works by adding tests

**Review by ArtikTico — COMMENTED** [review]
check my review

**Pull request merged**